core/domain/feedback_jobs_one_off.py

```diff
@@ -99,3 +99,27 @@ def _cache_last_nonempty_message_author_id(
             thread_model.last_nonempty_message_author_id = message_author_id
             return True
         return False
+
+
+class GeneralFeedbackThreadUserOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job for deleting GeneralFeedbackThreadUserModels with user_id
+    equal to None or 'None'.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        """Return a list of datastore class references to map over."""
+        return [feedback_models.GeneralFeedbackThreadUserModel]
+
+    @staticmethod
+    def map(model_instance):
+        """Implements the map function for this job."""
+        if model_instance.user_id == 'None' or model_instance.user_id is None:
```

Comment:
Separate these cases and yield separate values for each. I think it would be useful to see the breakdown.

Reply:
Done.